SparkPost/2web2ui
# FE-787 - Add Cypress integration tests for the Events feature

[FE-787](https://jira.int.messagesystems.com/browse/FE-787)

### What Changed
- Adds Cypress tests for Events list page and details page
- Parallelizes Cypress tests by breaking them up in to multiple groups using the `--group` flag and multiple Travis CI runners
- Removes remaining duplicate `/// <reference types="Cypress" />` for intelligent code completion in favor of setup file
- Updates page `<title>` for the Events list page and details page
- Replaces last remaining instances of local `stubRequest` functions with `cy.stubRequest()`

### How To Test
- Verify that Travis CI runs Cypress tests in two batches, reducing overall run time
- Verify that happy and unhappy paths fo the events page are adequately tested by Cypress

### To Do
- [x] Add tests for query params
- [x] Delete Jest tests duplicated by Cypress
- [x] Details Page - Add tests for message history clicking
- [x] Details Page - Add tests for disclaimer content rendering - **tested well with a Jest test**
- [x] Add tests for the orphaned events
